src/ImageSharp/Configuration.cs

```diff
@@ -34,6 +34,30 @@ public class Configuration
         private readonly List<IImageFormat> imageFormatsList = new List<IImageFormat>();
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="Configuration"/> class.
+        /// </summary>
+        public Configuration()
+            : this(false)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Configuration" /> class.
+        /// </summary>
+        /// <param name="autoloadWellknownFormats">if set to <c>true</c> [autoload wellknown formats].</param>
+        internal Configuration(bool autoloadWellknownFormats)
```

Comment:
:+1:  for the idea of having this "smart autoconfiguration"
I think it would be better to have this code in a  static `Configuration.CreateDefaultInstance()` method instead of a parametric base constructor.